src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -0,0 +1,346 @@
+        Implements IDisposable
+        Public Sub Dispose() Implements IDisposable.Dispose
+        End Sub
+    End Class
+    Sub Main(args As String())
+        Using c1 As New C'BIND:"Dim c1 As New C"
+            Console.WriteLine(c1)
+        End Using
+    End Sub
+End Module
+]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IUsingStatement (OperationKind.UsingStatement)
+  IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+    IVariableDeclaration: c1 As Program.C (OperationKind.VariableDeclaration)
+      Initializer: IObjectCreationExpression (Constructor: Sub Program.C..ctor()) (OperationKind.ObjectCreationExpression, Type: Program.C)
+  IBlockStatement (0 statements) (OperationKind.BlockStatement)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/17913"), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub UsingStatementDeclaration()
+            Dim source = <![CDATA[
+Module Program
+    Class C
+        Implements IDisposable
+        Public Sub Dispose() Implements IDisposable.Dispose
+        End Sub
+    End Class
+    Sub Main(args As String())
+        Using c1 As C = New C'BIND:"c1 As C = New C"
+            Console.WriteLine(c1)
+        End Using
+    End Sub
+End Module
+]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IUsingStatement (OperationKind.UsingStatement)
+  IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+    IVariableDeclaration: c1 As Program.C (OperationKind.VariableDeclaration)
+      Initializer: IObjectCreationExpression (Constructor: Sub Program.C..ctor()) (OperationKind.ObjectCreationExpression, Type: Program.C)
+  IBlockStatement (0 statements) (OperationKind.BlockStatement)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub ConstDeclaration()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main(args As String())
+        Const i1 As Integer = 1'BIND:"Const i1 As Integer = 1"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub ConstMultipleDeclaration()
```

Comment:
Can `Const` declarations in VB have `As New`? I think that's a compilation error.